_events/repeated/dei-working-group-call.md

```diff
@@ -2,22 +2,22 @@
 title: Diversity, Equity, and Inclusion Working Group Call
 subtitle:
 location: Virtual
-expires: 2022-07-31
-event_date: "every other Thursday at 15:00-16:00 Eastern"
+expires: 2022-12-31
+event_date: "every other Wednesday at 15:00-16:00 Eastern"
 layout: event
 category: dei
 time:
-  - - start: 2022-01-13 15:00 EST
+  - - start: 2022-08-10 15:00 EST
```

Comment:
I'm still not 100% sure I understand how the time zone stuff works, but the actual web page is showing 1:00 PM PT with the "add to calendar" button. I think if we put ET instead of EST, this should resolve the issue? Sorry I cannot verify myself this week as I'm a bit indisposed due to a conference.